### PR TITLE
Adds l2-transfer script

### PR DESCRIPTION
**Pull request by gigamesh**
Script for transferring ETH between L2 accounts (needed because Metamask has bugs preventing it using their extension: https://optimism-workspace.slack.com/archives/C01KXHPU3C6/p1624587229066300)

### CODE REVIEW

File: scripts/l2-transfer.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+const { providers, Wallet, utils, Contract } = require("ethers");
+const { getContractInterface } = require("@eth-optimism/contracts");
+const { JsonRpcProvider } = providers;
+
+const cfg = config();
+
+const provider = new JsonRpcProvider("https://mainnet.optimism.io");
+const wallet = new Wallet(cfg.privateKey).connect(provider);
+const l2ETHContract = new Contract(
+  "0x4200000000000000000000000000000000000006",
```
Comment (smartcontracts) [inline scripts/l2-transfer.ts]: Tip for future ref: you can import `predeploys` from `@eth-optimism/contracts` to get this address: `predeploys.OVM_ETH`

File: scripts/l2-transfer.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/**
+ * This script transfers ETH between Optimistic Mainnet accounts
+ */
+
+const { providers, Wallet, utils, Contract } = require("ethers");
+const { getContractInterface } = require("@eth-optimism/contracts");
+const { JsonRpcProvider } = providers;
+
+const cfg = config();
```
Comment (smartcontracts) [inline scripts/l2-transfer.ts]: Where does `config` get imported?

File: scripts/l2-transfer.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/**
+ * This script transfers ETH between Optimistic Mainnet accounts
+ */
+
+const { providers, Wallet, utils, Contract } = require("ethers");
```
Comment (smartcontracts) [inline scripts/l2-transfer.ts]: Since this is a `.ts` file I would use `import` instead of `require`

File: scripts/l2-transfer.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+(async () => {
+  const address = await wallet.getAddress();
+  console.log(`Sending from ${address}`);
+
+  const balance = await wallet.getBalance();
+  console.log(`Balance: ${utils.formatEther(balance.toString())}`);
+
+  console.log(`Sending to ${cfg.to}`);
+
+  const tx = await l2ETHContract.transfer(cfg.to, utils.parseUnits(cfg.amount));
```
Comment (smartcontracts) [inline scripts/l2-transfer.ts:34]: You should be able to do this with normal transactions:

`await wallet.sendTransaction({ to: cfg.to, value: ... })`

File: scripts/l2-transfer.ts (hunk 2 above)
Comment (smartcontracts) [reply scripts/l2-transfer.ts]: I see, it's defined below. I'd recommend defining it above the main function and calling config within the main function.

File: scripts/l2-transfer.ts (hunk 4 above)
Comment (gigamesh) [reply scripts/l2-transfer.ts:34]: That way resulted in "cannot estimate gas" error. I'm assuming I need to specify the gas limit? 
Is there a strong reason to do it that way vs instantiating the contract?

**Review by smartcontracts — APPROVED** [review]
Lgtm

**Pull request merged**